include/net-snmp/library/tools.h

```diff
@@ -290,6 +290,8 @@ extern          "C" {
     NETSNMP_IMPORT
     int             netsnmp_string_time_to_secs(const char *time_string);
 
+    char            *netsnmp_gethomedir();
```

Comment:
NETSNMP_IMPORT should be added to make this function available to apps when building libsnmp as a Windows DLL.

Reply:
It wasn't used outside the lib. Now it is, added.